backend/main.py

```
@@ -107,38 +111,127 @@ async def startup_event():
-        logger.warning("⚠️  Auto-setup failed, continuing with manual checks...")
+    # Run database migrations in production
+    if settings.environment == "production":
+        try:
+            logger.info("🗄️  Running database migrations...")
+            import subprocess
+            import sys
+            from pathlib import Path
+            
+            # Change to database directory and run migrations
+            db_dir = Path(__file__).parent / "database"
+            result = subprocess.run(
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)